functests/5_latency_testing/latency_testing.go

```diff
@@ -92,16 +92,16 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 		if err != nil {
 			fmt.Println(err.Error())
```

Comment:
what about handling an error here?

Reply:
correcting it ..
, thank you!

Reply:
@cynepco3hahue  actually we can't use gomega Expect on the error here , because we'll be getting an error in the negative checks & that seems to be resulted in the error,& if we use Expect on that error , we'll end up getting the gomega truncated error. We should not fail in case of negative checks.So I will add this expectation under a condition.

Reply:
I think you can just remove the check for err here, if it is an error it will fail under the positive test check